proof/crefine/ARM_HYP/CSpace_C.thy

```diff
@@ -152,11 +153,7 @@ lemma Arch_maskCapRights_ccorres [corres]:
    apply (cases arch_cap)
        by (fastforce simp add: cap_get_tag_isCap isCap_simps  simp del: not_ex simp_thms(44))+
 
-(* FIXME: move to Wellformed_C (or move to_bool_bf out of Wellformed_C) *)
-lemma to_bool_mask_to_bool_bf:
-  "to_bool (x && 1) = to_bool_bf (x::word32)"
-  by (simp add: to_bool_bf_def to_bool_def)
-
+(* FIXME: move to Wellformed_C *)
```

Comment:
I wasn't sure if this FIXME was originally just about this generic `to_bool_bf` lemma or if it also included the following more seL4 specific lemmas. For now I've left it in, it would probably be nice to move the latter lemmas, although they're too specific to go in the new `Boolean_C` file.

Reply:
I don't think `capAllowRead_CL` is available in `Wellformed_C` as it comes from parsing the kernel, so this FIXME no longer makes sense and I think can be removed.
Your moving both the lemma and to_bool_bf to Boolean_C I think addresses the issue sufficiently.

Reply:
I thought about this some more. Leaving this FIXME for now is fine.